src/gsy_e/models/strategy/__init__.py

```diff
@@ -1,3 +1,4 @@
+# pylint: disable=too-many-lines
```

Comment:
This module has only be changed by Black because I fixed one method interface. You can ignore reviewing this. Thanks